src/components/MediaQuerryDebugger/index.js

```diff
@@ -0,0 +1,28 @@
+import React, { useState } from 'react'
+import classNames from 'classnames/bind'
+import propTypes from 'prop-types'
+import styles from './styles.module.scss'
+
+const MediaQueryDebugger = ( props ) => {
+  
+  if (props.isVisible != 'true') return null
```

Comment:
This was previously process.env.GATSBY_DEBUG_MEDIA_QUERIES !== 'true'. It was replaced to resolve an eslint error that was blocking the build pipeline.